For whatever reason some perl nonsense is nuking our
linkage against the perl swig library. That results
in all the native modules failing to resolve many
symbols. Shove it in LDFLAGS where it cannot be touched
as a workaround.

diff --git a/subversion/bindings/swig/perl/native/Makefile.PL.in b/subversion/bindings/swig/perl/native/Makefile.PL.in
index 3b36901..984b38f 100644
--- a/subversion/bindings/swig/perl/native/Makefile.PL.in
+++ b/subversion/bindings/swig/perl/native/Makefile.PL.in
@@ -90,9 +90,9 @@ my %config = (
                   (map {"-L$_"} @ldpaths),
                   @ldmodules, '-lsvn_swig_perl-1',
                   `$swig -perl -ldflags`)],
-    LDFLAGS => join(' ', (map {"-L$_"} @ldpaths), '@LDFLAGS@ @SWIG_PL_LINK@'),
+    LDFLAGS => join(' ', (map {"-L$_"} @ldpaths), '@LDFLAGS@ -lsvn_swig_perl-1 @SWIG_PL_LINK@'),
     LDDLFLAGS => join(' ', (map {"-L$_"} @ldpaths),
-                      '@LDFLAGS@ @SWIG_PL_LINK@', $Config{lddlflags}),
+                      '@LDFLAGS@ -lsvn_swig_perl-1 @SWIG_PL_LINK@', $Config{lddlflags}),
     test => { TESTS => "$swig_srcdir/perl/native/t/*.t" }
 );
 
